app/tests/spec/models/auth_brokers/fx-ios-v1.js

```diff
@@ -118,6 +119,24 @@ define(function (require, exports, module) {
         });
       });
 
+      describe('afterCompleteSignInWithCode', () => {
+        let account;
+
+        beforeEach(() => {
+          sinon.spy(broker, 'afterCompleteSignInWithCode');
+          sinon.spy(broker, '_notifyRelierOfLogin');
+          sinon.spy(FxDesktopV1AuthenticationBroker.prototype, 'afterCompleteSignInWithCode');
```

Comment:
Since you are modifying a prototype method, there could be interference with tests that run afterwards. Can you use a [sinon sandbox](https://sinonjs.org/releases/v2.1.0/sandbox/) and do a `sandbox.restore` in afterEach to avoid any cross-test contamination?